externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/shim/FlinkSessionManager.scala

```diff
@@ -21,26 +21,17 @@ import org.apache.flink.table.gateway.api.session.{SessionEnvironment, SessionHa
 import org.apache.flink.table.gateway.service.context.DefaultContext
 import org.apache.flink.table.gateway.service.session.Session
 
-import org.apache.kyuubi.engine.flink.FlinkEngineUtils.FLINK_RUNTIME_VERSION
 import org.apache.kyuubi.util.reflect._
 import org.apache.kyuubi.util.reflect.ReflectUtils._
 
 class FlinkSessionManager(engineContext: DefaultContext) {
```

Comment:
IIRC this class was introduced as an adapter, is it possible to remove it now?

Reply:
Thanks for your suggestion! I will try to remove this class.